go.mod

```diff
@@ -103,6 +103,7 @@ require (
 	github.com/inconshreveable/mousetrap v1.0.0 // indirect
 	github.com/josharian/intern v1.0.0 // indirect
 	github.com/json-iterator/go v1.1.12 // indirect
+	github.com/kulti/thelper v0.5.1 // indirect
```

Comment:
I am not sure why `thelper` has come here as an indirect dependency. Try removing that and run `go mod tidy`